packages/apps-v5/src/commands/ps/type.js

```diff
@@ -14,8 +14,39 @@ https://devcenter.heroku.com/articles/procfile`)
 async function run(context, heroku) {
   let app = context.app
 
+  // will remove this flag once we have
+  // successfully launched larger dyno sizes
+  let isLargerDyno = false
```

Comment:
All of the comments from the ps/scale file apply here as well

Reply:
My responses above are the same for `ps/scale`.